src/components/search/SearchResults.svelte

```diff
@@ -195,56 +198,56 @@ li {
 }
 
 .focused {
-    background-color: var(--bg-gray-01);
+  background-color: var(--bg-gray-01);
 }
 
 </style>
 
 <svelte:window bind:innerWidth={windowWidth} on:keydown={handleKeypress} />
 
 <Portal>
-{#if $store.searchIsActive && $store.searchQuery.length}
-  <div
-  id="telemetry-search-results"
-  style="left: calc({x}px + var(--space-base)); width: calc({width}px - var(--space-base));"
-    transition:fly={{ duration: 20, y: -10 }}
-    class="telemetry-results">
-      <div class="header-container">
-          {#if $searchResults.total}
-          <div class="header header--loaded" in:fly={{ x: -5, duration: 200 }}>
-              <div>found {$searchResults.results.length} of
-                  {formatTotal($searchResults.total)} probes
-              </div>
-          </div>
-          {:else}
-          <div class="header" out:fly={{ x: 5, duration: 200 }}>
-              <LineSegSpinner color={'var(--subhead-gray-02)'} />
-              <div>
-                  getting the probes – one second!
-              </div>
-          </div>
+  {#if searchIsActive && query.length}
+    <div
+      id="telemetry-search-results"
+      style="left: {x}px; width: {width}px;"
+      transition:fly={{ duration: 20, y: -10 }}
+      class="telemetry-results">
+        <div class="header-container">
+          {#if results.length}
```

Comment:
I think I mentioned this in the previous PR, but it feels like there should be another case here, where if we are either waiting for the throttle timer to finish or we're waiting for a result set, where we say `fetching probes` + a spinner or something like that. Right now, I type and nothing immediately happens, and it feels like a step back in terms of experience.